src/components/indicatorMenu/IndicatorMenu.vue

```diff
@@ -5,7 +5,12 @@
                 {{ appConfig.indicators[selectedIndicator].humanReadableName }}
             </p>
             <v-icon :icon="menuOpen ? 'mdi-chevron-down' : 'mdi-chevron-up'" end></v-icon>
-            <v-menu activator="parent" v-model="menuOpen">
+            <v-menu
+                activator="parent"
+                v-model="menuOpen"
+                :close-on-content-click="false"
+                :close-on-back="false"
+            >
```

Comment:
It seems the click away to close does work only when you click away above the menu, but not when you click away to the right of the menu.

Reply:
Good spot! This was because the vuetify-generated overlay content div under the menu was invisibly hogging most of the screen width and not responding to the click event. Putting the max width on that rather than the v-list fixes the issue.